plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/messages.properties

```diff
@@ -0,0 +1,3 @@
+installJobName=Installing the Cloud SDK...
+startModifying=Installing/upgrading the Cloud SDK... (may take several minutes)
```

Comment:
I don't think you need this message, at least not yet

Reply:
Removed.